### PR TITLE
Ensure that command result should be reported even when the handling context's deadline passes

### PR DESCRIPTION
**Pull request by nghialv**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/app/piped/planpreview/handler.go [hunk 1]
```diff
@@ -262,6 +261,7 @@ func (h *Handler) handleCommand(ctx context.Context, cmd model.ReportableCommand
 		if err != nil {
 			logger.Error("failed to marshal command result", zap.Error(err))
 		}
+
 		if err := cmd.Report(ctx, model.CommandStatus_COMMAND_FAILED, nil, output); err != nil {
```
Comment (nghialv) [inline pkg/app/piped/planpreview/handler.go:265]: With the current code, this `Report` function will not be executed when the command context's deadline passes.

Comment (khanhtc1202) [reply pkg/app/piped/planpreview/handler.go:265]: Ah, got it. Nice catch 👍

**Comment by khanhtc1202** [comment]
🚀 
/approve

**Pull request merged**